docs/blog/2020-04-16-gatsby-survey-goes-command-line/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Gatsby's First Survey Goes Command Line (And here is what users have to say so far)"
+date: 2020-04-16
+author: Shannon Soper
+excerpt: "When we wanted to survey the community about ways we can make Gatsby better, the command line just felt like the right place to ask. Here's what you told us. (And if you didn't get a chance to respond, we'd still love to hear from you)."
+tags:
+  - community
+  - diversity-and-inclusion
+  - gatsby-cloud
+---
+
+Some of you have recently seen a link in the CLI, inviting you to take a four question survey about Gatsby:
+
+![Invitation to take Gatsby user survey from the command line](./survey-screenshot.png)
+
+Thank you to all who responded to the survey; myself and other engineers have personally read all the responses and are summarizing them in this blogpost, in an anonymized manner!
+
+If by chance you have not yet taken the survey, we definitely still want to hear from you! You can submit your own survey response today by upgrading your version of gatsby in a project’s `package.json` file and rerunning dependency and install commands. Or you can install the Gatsby CLI using `npm install -g gatsby-cli` (see the [Quickstart Guide](https://www.gatsbyjs.org/docs/quick-start/) for more details). To trigger the survey, you can run `gatsby feedback` at any time. Otherwise, you’ll be prompted every 3 months, at the most.
+
+(If you’d like to opt out, visit https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out).
```

Comment:
```suggestion
(If you’d like to opt out, visit [CLI Feedback: How to opt out](/docs/CLI-feedback/#how-to-opt-out)).
```
i don't know it is ok to change to this link text to get the local domain removed?

Reply:
My understanding is that typing out full URLs is viewed as a style choice by the writer, and so either way is fine. Since Shannon put it in this way I left it alone.